tests/expressions_case/tests.py

```diff
@@ -722,6 +722,22 @@ def test_filter_with_aggregation_in_predicate(self):
             transform=itemgetter("integer", "integer2", "max"),
         )
 
+    def test_case_with_extra_kwargs(self):
+        case_expression = Case(
+            When(integer=1, then=Value(10)),
+            When(integer=2, then=Value(20)),
+            default=Value(0),
+            template="CASE %(cases)s ELSE %(default)s + 5 END",
+        )
+        self.assertListEqual(
+            list(
+                CaseTestModel.objects.annotate(values=case_expression).values_list(
+                    "values", flat=True
+                )
+            ),
+            [10, 20, 5, 20, 5, 5, 5],
+        )
```

Comment:
The other tests in this file seem to favor `assertQuerySetEqual`, so I suggest:
```suggestion
        self.assertQuerySetEqual(
            CaseTestModel.objects.annotate(values=case_expression).order_by("pk"),
            [10, 20, 5, 20, 5, 5, 5],
            transform=attrgetter("values"),
        )
```